src/main/java/com/smartnews/jpa_entity_generator/util/TypeConverter.java

```diff
@@ -18,8 +18,8 @@ public static String toJavaType(int typeCode) {
                 return "Long";
             // case Types.BINARY:
             case Types.BIT:
-                // return "Boolean";
-                return "boolean";
+                // my sql
+                return "Boolean";
```

Comment:
This can be a breaking change. If I remember correctly, setting primitive type here means the value will be `false` if absent. Is there any reason you changed this?

Reply:
When the database has a **nullable** column, primitive type field of the entity can't store **null**, NPE will occur. Wrapped Type can represent true/false and **not set**

Reply:
Assuming we force make all boolean feild to be ```not null```.
There is an another situation:

Sometimes, we use ExampleMatcher of JPA, and telling JPA ignore the null value field:
```
ExampleMatcher.matchingAll().withIgnoreNullValues()
```
or mybatis :
```
<if test="aNullableBooleanState != null">
a_nullable_boolean_state= #{aNullableBooleanState, jdbcType="BIT"}
</if>
```

If the field type is primitive, we will always query data with and ```field =true/false```, can't leave with this condition.

Reply:
@slankka 
I'm terribly sorry for my belated response here. 

That makes sense. That said, I'd like to avoid bringing any breaking changes to this library. This library is widely used at SmartNews and I am no longer working for the company.

If you're still interested in working this PR and have time for it, could you introduce a flag in the configuration allowing users to choose `Boolean` over the primitive type plus set it as false by default?